profiles/logical/holo/hydra/master/default.nix

```
@@ -109,6 +115,17 @@ in
     };
   };
 
+  services.restic.backups = {
+    remotebackup = {
+      user = "root";
```

`root` is the default, but it doesn't have to be that wide.

```suggestion
      user = "postgres";
```